### PR TITLE
Some simple additions to the documentation

### PR DESCRIPTION
**Pull request by marcharper**
Follow-ups from Tuesday's discussions.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -5,12 +5,16 @@ All contributions to this repository are welcome via pull request on the `github
 
 The project follows the following guidelines:
 
-1. Use the base Python library unless completely necessary.
+1. Use the base Python library unless completely necessary. Numpy is ok.
```
Comment (drvinceknight) [inline docs/contributing.rst]: Is it worth wording this more along the lines of: please use what is currently in `requirements.txt`, if you need to use an external library perhaps discuss this with us? (Or maybe get rid of this completely?)

**Comment by drvinceknight** [comment]
This is great, just my one comment about the first guideline. Not sure it's still worth insisting on... Happy to go with whatever you feel is best.

**Comment by marcharper** [comment]
Yeah I thought about that... how's this change? I just wanted to mention numpy specifically since we are starting to use it more.

**Comment by drvinceknight** [comment]
> Yeah I thought about that... how's this change? I just wanted to mention numpy specifically since we are starting to use it more.

Yeah great! 

This is good to go (have already said so on the gitter channel) :+1:

**Pull request merged**